examples/schelling.jl

```diff
@@ -97,7 +97,7 @@ function initialize(; numagents = 320, griddims = (20, 20), min_to_be_happy = 3,
     rng = Random.MersenneTwister(seed)
     model = ABM(
         SchellingAgent, space;
-        properties, rng, scheduler = Schedulers.randomly
+        properties, rng, scheduler = Schedulers.randomly()

```

Comment:
Unfortunately this is a **massively breaking change** that we simply cannot do without releasing 6.0. I'd recommend to make a new scheduler `Randomly`.

Reply:
(and similarly for all other schedulers that need to change)

Reply:
Isn't that still breaking? Or should the old schedulers also be kept, with something like a warning to use the new ones?

Reply:
Oh yes, sorry I wasn't clear. ALl old schedulers need to be kept exactly as they are :D 

Warning would be nice, sure.